### PR TITLE
Updating primer pages 

### LINKED ISSUE #416: Update homepage text and Primer documentation to ensure it is aligned with latest Open Ownership research and style guide
Revisit the following pages to check alignment with latest Open Ownership research and style guide. Where possible, link to OO tools, research or content for further information:

- [x] [Homepage](https://standard.openownership.org/en/0.2.0/index.html) - flesh out with more detail like [OCDS landing page](https://standard.open-contracting.org/latest/en/)?
- [x] [Primer](https://standard.openownership.org/en/0.2.0/primer/index.html)
- [x] [What is beneficial ownership?](https://standard.openownership.org/en/0.2.0/primer/whatisbo.html)
- [x] [What is the Beneficial Ownership Data Standard?](https://standard.openownership.org/en/0.2.0/primer/whatisthebods.html)

Discuss how to update BODS documentation to best flag/showcase connections with the [Open Contracting Data Standard](https://standard.open-contracting.org/latest/en/guidance/map/beneficial_ownership/) and to thematic areas of interest to users like procurement, anti-corruption, banking, tax, etc...

### PR DESCRIPTION
**Pull request by kathryn-ods**
# Overview

- What does this pull request do? Updates the primer pages
- How can a reviewer test or examine your changes? Build the docs and look at the primer pages 
- Who is best placed to review it? @kd-ods @StephenAbbott 

Closes #416

## Translations

- [ ] New or edited strings appearing as a result of this PR will be picked up for translation
- [ ] I've notified the translation coordinator of any new strings that will need
      translating. See: https://openownership.github.io/bods-dev-handbook/translations.html

## Documentation & Release

- [x] I've checked that the documentation builds without failing. See: https://github.com/openownership/data-standard#building-the-documentation-locally
- [ ] I've thought about how and when this needs to be released. See:
      https://openownership.github.io/bods-dev-handbook/standard_releases.html      
- [ ] I've updated the changelog, if necessary.
- [ ] I've updated [reference.rst](https://standard.openownership.org/en/latest/schema/reference.html) to reflect any changes to schema structure or naming.
- [ ] I've added or edited any other related documentation. See: https://github.com/openownership/bods-dev-handbook/blob/master/style_guide.md

### CODE REVIEW

**Comment by kathryn-ods** [comment]
note:
- some links aren't working due to those pages needing work first 
- I've removed https://standard.openownership.org/en/latest/primer/index.html and put both pages into one with subheadings in the sidebar (like https://standard.openownership.org/en/latest/examples/index.html) 

I think the primer index page isn't needed now that we've slimmed the primer down so much. I'm undecided on whether having the 2 pages together is good or they should be fully split out - I can switch that around if needed.

File: docs/primer/index.rst [hunk 1]
```diff
@@ -1,17 +1,57 @@
...
+
+* improves the functionality of beneficial ownership information
+* reduces the cost of producing, using, and maintaining beneficial ownership information
+* has a greater chance of meeting beneficial ownership transparency policy goals than unstructured data
+
+Without common standards, combining and analysing data from different sources can be expensive and time-consuming, making it more difficult to understand beneficial ownership networks.
+
+BODS allows publishers to provide statements that describe:
+
+* :any:`details about natural persons <schema-person-record>`
```
Comment (kd-ods) [inline docs/primer/index.rst:51]: The links in this block of bullets will work with the following changes:

```

* :any:`details about natural persons <schema-person-record>`
* :any:`details about entities <schema-entity-record>`
* :any:`identifiers for natural persons and entities <guidance-identifiers>`
* :any:`types of interests that can be held in entities <schema-interest>`
* :any:`provenance information <schema-source>`
* :any:`historical, as well as current, information <representing-updates>`
* :any:`placeholders, when information must be witheld <schema-interested-party>`
```
BUT - I don't think it's necessarily helpful to readers to include these links. They lead to very specific bits of the standard which are unlikely to be that useful if someone is at the stage of reading the primer. I'd rather they went exploring via the other links in this section.

**Review by kd-ods — CHANGES_REQUESTED** [review]
I've provided some fixes, thoughts, and suggestions.

File: docs/primer/index.rst (hunk 1 above)
Comment (kathryn-ods) [reply docs/primer/index.rst:51]: Yes that's a fair point - it might be overwhelming to have this many links

**Comment by kathryn-ods** [comment]
@kd-ods have accepted those changes

**Review by kd-ods — APPROVED** [review]
This is good to go :-)

**Pull request merged**